### PR TITLE
Added release notes for v2.12.0.

### PR DESCRIPTION
**Pull request by JamesKovacs**

### CODE REVIEW

**Review by rstam — APPROVED** [review]
LGTM

File: Release Notes/Release Notes v2.12.0.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+
+* Support for Hidden Indexes in MongoDB 4.4
+* Support for AWS temporary credentials in client-side field level encryption (CSFLE)
+* Support for Azure and GCP keystores in client-side field level encryption (CSFLE)
+* Support for client-side field level encryption (CSFLE) on Linux and Mac OSX
+* Support for GSSAPI/Kerberos on Linux
+* Various improvements in serialization performance
+* Fixed DNS failures in Kubernetes and Windows Subsystem for Linux (WSL/WSL2)
+* Fixed memory leak in heartbeat when cluster is inaccessible
+* Fixed SDAM deadlock when invalidating former primary
```
Comment (DmitryLukyanov) [inline Release Notes/Release Notes v2.12.0.md:15]: Add .NET Standard 2.1 support ?

**Review by DmitryLukyanov — APPROVED** [review]
Minor comment and then LGTM

File: Release Notes/Release Notes v2.12.0.md (hunk 1 above)
Comment (JamesKovacs) [reply Release Notes/Release Notes v2.12.0.md:16]: I debated adding this or not as it opens us up for questions about whether .NET 5.0 and 6.0 are supported. I added mention of .NET Standard 2.1 support and we can field questions if they come up.

**Pull request merged**